db/seeds.rb

```diff
@@ -5,8 +5,40 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-words = %w(in this case there is no need to guess how many requests can be in an application that might finish in the future in some cases ajax requests can be part of a functions inner logic which can be quite complicated and in that case you might not wait until said)
+
+new_category = Category.create(name: "animals")
+
+words = %w(owl yak lion dog bear cat bird horse zebra hippo monkey gorilla fish reptile frog snake lizard iguana rhino bee eagle fox gopher kangaroo spider jellyfish shrimp chickadee swan bison buffalo turtle tortoise deer armadillo quail squirrel giraffe goose vulture groundhog penguin seal roadrunner reindeer shark skunk sloth woodchuck)
 
 words.each do |word|
-  Word.create(text: word, points: word.length * 100, x: rand(50..400), y: 0)
+  if word.length <= 4
+    new_word = Word.new(text: word, points: word.length * 100, x: rand(25..400), y: 0, level: 1)
+    new_word.category = new_category
+    new_word.save
```

Comment:
This could all be in one line.
`new_category.words.create(text: word, points: word.length * 100, x: rand(25..400), y: 0, level: 1)`